utils/dataset_manifest/create.py

```diff
@@ -56,18 +46,22 @@ def main():
                 data_dir = source.split(items[position])[0]
             except Exception as ex:
                 sys.exit(str(ex))
-            sources = list(filter(_is_image, glob(source, recursive=True)))
+            sources = list(filter(is_image, glob(source, recursive=True)))
+
+        sources = list(filter(lambda x: 'related_images{}'.format(os.sep) not in x, sources))
+        related_images = detect_related_images(sources, source)
+        meta = { k: {'related_images': related_images[k] } for k in related_images }
         try:
             assert len(sources), 'A images was not found'
             manifest = ImageManifestManager(manifest_path=manifest_directory)
-            meta_info = manifest.prepare_meta(sources=sources, is_sorted=False,
+            meta_info = manifest.prepare_meta(sources=sources, meta=meta, is_sorted=False,
```

Comment:
`source` maybe `glob` pattern. In this case does not work correctly.

Reply:
Looks like it was fixed.
At least this run works fine:
```
python create.py --output-dir ~ "~/cvat/../cvat/./share/**/*"
```

Reply:
`create.py` supported  `*`, `?`, `[]` characters. What about `?`, `[]` support with related images?
`data_dir` can not be used in this case?

Reply:
I didn't see such examples in README.md, but now I've seen that there is mention in the text. 
Anyway, I suppose there is not a problem to support them the same way like ``*``

Reply:
Updated, tested with ``python create.py --output-dir ~ "~/cvat/../c[v]a?/./share/**/*.jpg"`` and a couple of other commands.